mmv1/templates/terraform/examples/integration_connectors_endpoint_attachment.tf.erb

```diff
@@ -6,4 +6,5 @@ resource "google_integration_connectors_endpoint_attachment" "<%= ctx[:primary_r
   labels = {
     foo = "bar"
   }
+  endpoint_global_access = false
```

Comment:
Since you mentioned the default is `false`, can this line be removed from the example?

Reply:
Done